src/Plugin/views/field/EntityBundle.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce\Plugin\views\field\EntityBundle.
+ */
+
+namespace Drupal\commerce\Plugin\views\field;
+
+use Drupal\views\Plugin\views\field\Field;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Field handler to display the entity bundle. It displays nothing if there's
```

Comment:
Docblocks like this should have a short single line description, then an empty line, then the full description.

"Displays the entity bundle.

Can be configured to show nothing when there's only one possible bundle, allowing the 'Hide empty column' table setting to hide the column."